src/components/DatePicker/DatePicker.vue

```diff
@@ -102,6 +104,11 @@ export default class DatePicker extends Vue {
     return this.$refs.form.validate()
   }
 
+  /** Returns whether date validation passes. */
+  public isDateValid (): boolean {
+    return this.$refs.dateTextField.valid
```

Comment:
You should use conditional chaining in case `dateTextField` is undefined (ie, before the component is initially rendered).

Reply:
Please add a unit test or 2 to verify this this returns True/False as expected.

Reply:
will get that updated.

Reply:
and yes, tests too.  keeping me honest....thx

Reply:
updated to use conditional chaining and ended up adding a bunch of tests

![image](https://user-images.githubusercontent.com/6685223/143152492-01f3c1c7-922f-4afe-8503-d3496a82537b.png)